README.md

```diff
@@ -213,6 +213,9 @@ effects, reference the following snippet for enabling OpenSL ES:
     
     // Check if OpenSL ES is disabled 
     WebRtcAudioUtils.deviceIsBlacklistedForOpenSLESUsage()
+
+### Known Issues
+- Using the WebRTC based acoustic echo canceler on the Pixel 3 XL will disable the microphone system wide.
```

Comment:
Provide a code snippet of the API calls that results in this behavior. Also add a link to the bug you filed in WebRTC